docs/source/conf.py

```diff
@@ -289,5 +296,8 @@
 #texinfo_no_detailmenu = False
 
 intersphinx_mapping = {
-    'ipython': ('http://ipython.org/ipython-doc/dev/', None)
+    'ipython': ('http://ipython.org/ipython-doc/dev/', None),
+    'ipykernel': ('http://ipython.org/ipykernel-doc/dev/', None),
+    'nbconvert': ('http://jupyter.org/nbconvert-doc/dev/', None),
+    'nbformat': ('http://jupyter.org/nbformat-doc/dev/', None)
```

Comment:
nbformat docs are at http://nbformat.readthedocs.org/en/latest/ . nbconvert docs will probably be on RTD, but I don't think they're built yet. And ipykernel may not need its own docs at all.